### PR TITLE
rke2: fix master etcd checks

### PR DESCRIPTION
**Pull request by vardhaman22**
issues: https://github.com/rancher/cis-operator/issues/350 https://github.com/rancher/rancher/issues/46812
1. fixed below changes in all rke2 profiles
- added 1.1.11 and 1.1.12 missing checks in the profiles
- updated remediation for 1.1.7, 1.1.8, 1.1.11 for all the cis profiles
- updated scored to false for 1.1.7,1.1.8 and 1.1.11 checks
2. updated 1.1.12 for rke2 cis 1.23,1.24 hardened profiles, updated remediation and set scored to false also removed type:skip from hardened profiles wherever present.

### CODE REVIEW

File: package/cfg/rke2-cis-1.8-permissive/master.yaml [hunk 1]
```diff
@@ -160,6 +164,39 @@ groups:
...
+        audit: "stat -c %U:%G $etcddatadir"
+        tests:
+          test_items:
+            - flag: "etcd:etcd"
+        remediation: |
+          On the etcd server node, get the etcd data directory, passed as an argument --data-dir,
+          from the command 'ps -ef | grep etcd'.
+          Run the below command (based on the etcd data directory found above).
+          For example, chown etcd:etcd /var/lib/etcd
+        scored: true
```
Comment (dereknola) [inline package/cfg/rke2-cis-1.8-permissive/master.yaml:198]:
```suggestion
        text: "Ensure that the etcd data directory ownership is set to etcd:etcd (Manual)"
        audit: "stat -c %U:%G $etcddatadir"
        tests:
          test_items:
            - flag: "etcd:etcd"
        remediation: |
          On the etcd server node, get the etcd data directory, passed as an argument --data-dir,
          from the command 'ps -ef | grep etcd'.
          Run the below command (based on the etcd data directory found above).
          For example, chown etcd:etcd /var/lib/etcd
        scored: false
```

**Review by dereknola — COMMENTED** [review]
My comment applies to all permissive profiles.

File: package/cfg/rke2-cis-1.8-permissive/master.yaml (hunk 1 above)
Comment (vardhaman22) [reply package/cfg/rke2-cis-1.8-permissive/master.yaml]: updated all the profiles.
Also removed `type: skip `for this check in hardened profiles wherever present.

**Review by andypitcher — APPROVED** [review]
LGTM

**Review by dereknola — APPROVED** [review]

**Pull request merged**